bodhi/util.py

```diff
@@ -435,12 +435,10 @@ def request2html(context, request):
 def update2html(context, update):
     request = context.get('request')
 
-    if hasattr(update, 'title'):
-        title = update.title
-    else:
-        title = update['title']
+    title = update.title
+    alias = update.alias
 
-    url = request.route_url('update', id=title)
+    url = request.route_url('update', id=alias)
```

Comment:
There have been bugs in the past where alias was `None`. Hopefully that issue is fixed, but we probably still want the `alias or title` logic in there.